organization.go

```diff
@@ -71,8 +71,9 @@ var ErrInvalidOrgFilter = &Error{
 
 // OrganizationFilter represents a set of filter that restrict the returned results.
 type OrganizationFilter struct {
-	Name *string
-	ID   *ID
+	Name   *string
+	ID     *ID
+	UserID *ID
```

Comment:
Should this be added to documentation somewhere if it's the API user who sets it? (Or, if it's only used internally, should it be set internally somewhere?)

Reply:
I will add that. good find!